lib/Reverse.ark

```diff
@@ -0,0 +1,13 @@
+{
+    #made by Natendrtfm (on Github) with <3
+    (let reverse (fun (inputed){
+        (assert (!= inputed "") "Argument length must be superior to 0") # Sounds weird to me , but it work ! 
```

Comment:
why should it be weird?
also you should just do a (if (= (len inputed) 0) "" { rest of the code }) because reversing "" should be possible

Reply:
i was tired at this moment 
i think the condition would work with `=` instead of `!=`